Readme.md

```diff
@@ -84,27 +70,118 @@ loadImage('examples/images/lime-cat.jpg').then((image) => {
 })
 ```
 
-## Know issues
+## Documentation
+
+This project is an implementation of the Web Canvas API and implements that API as closely as possible. For API documentation, please visit [Mozilla Web Canvas API](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API). (See [Compatibility Status](https://github.com/Automattic/node-canvas/wiki/Compatibility-Status) for the current API compliance.) All utility methods and non-standard APIs are documented below.
+
+### Utility methods
+
+* [createCanvas()](#createcanvas)
+* [createImageData()](#createimagedata)
+* [loadImage()](#loadimage)
+* [registerFont()](#registerfont)
+
+### Non-standard APIs
+
+* [Image#src](#imagesrc)
+* [Image#dataMode](#imagedatamode)
+* [Canvas#toBuffer()](#canvastobuffer)
+* [Canvas#createPNGStream()](#canvascreatepngstream)
+* [Canvas#createJPEGStream()](#canvascreatejpegstream)
+* [Canvas#createPDFStream()](#canvascreatepdfstream)
+* [Canvas#toDataURL()](#canvastodataurl)
+* [CanvasRenderingContext2D#patternQuality](#canvasrenderingcontext2dpatternquality)
+* [CanvasRenderingContext2D#filter](#canvasrenderingcontext2dfilter)
+* [CanvasRenderingContext2D#textDrawingMode](#canvasrenderingcontext2dtextdrawingmode)
+* [CanvasRenderingContext2D#globalCompositeOperator = "saturate"](#canvasrenderingcontext2dglobalcompositeoperator--saturate)
+* [CanvasRenderingContext2D#antialias](#canvasrenderingcontext2dantialias)
+
+### createCanvas()
+
+> ```ts
+> createCanvas(width: number, height: number, type?: "PDF"|"SVG") => Canvas
+> ```
+
+Creates a Canvas instance. This method works in both Node.js and Web browsers, where there is no Canvas constructor. (See `browser.js` for the implementation that runs in browsers.)
+
+```js
+const {createCanvas} = require('canvas')
```

Comment:
Standard 12 is enforcing `{ createCanvas }` (spacing), could we change to that? ☺️